semantic-conventions/src/tests/semconv/templating/test_markdown.py

```diff
@@ -36,7 +35,8 @@ def testRef(self):
         renderer._render_single_file(content, md, output)
         with open(self.load_file("markdown/ref/expected.md"), "r") as markdown:
             expected = markdown.read()
-        self.assertEqualWithDiff(expected, output.getvalue())
+
+        assert output.getvalue() == expected
```

Comment:
I _almost_ refactored this whole class, but I settled for just using pytest style assertions, which give much more usable messages (in my opinion). Luckily, it plays well with unittest. :)